components/NftCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { useConnection } from '@solana/wallet-adapter-react'
+import { Link, Button, Text, IconButton } from '@primer/react'
+import { FC } from 'react'
+import { LinkExternalIcon } from '@primer/octicons-react'
+
+const NftCard: FC<{ title: string; imageLink: string; hash?: string }> = ({ title, imageLink, hash }) => {
+    const { connection } = useConnection()
+    return (
+        <div
+            className='d-flex flex-column flex-justify-center flex-items-center border rounded-3 gap-2 py-2'
+            style={{ width: '185px' }}
+        >
+            <Text as='p' fontWeight='bold'>
+                {title}
+            </Text>
+            <img className='border rounded-3' style={{ height: '168px', width: '168px' }} src={imageLink} />
```

Comment:
I just saw two more things :eyes:  one is the `alt` attribute :sweat_smile:  and the other imo, it might be better to put the image in a wrapper `div` that has the size and adjust the image to it

Reply:
```suggestion
            <div className='border rounded-3 overflow-hidden d-flex flex-justify-center flex-items-center' style={{ height: '168px', width: '168px' }}>
                <img src={imageLink} alt={title} />
            </div>
```
![image](https://user-images.githubusercontent.com/72078237/179244264-4b707ca6-394a-4852-b652-66866dfaa54a.png)
![image](https://user-images.githubusercontent.com/72078237/179244801-438bc212-15a4-43a4-8a83-15f2ce16e18b.png)